R/stat-bin2d.r

```diff
@@ -80,15 +80,19 @@ StatBin2d <- proto(Stat, {
 
     # Determine breaks, if omitted
     if (is.null(breaks)) {
-      breaks <- list(
-        seq(origin[1], max(range$x) + binwidth[1], binwidth[1]),
-        seq(origin[2], max(range$y) + binwidth[2], binwidth[2])
-      )
-    } else {
-      stopifnot(is.list(breaks))
-      stopifnot(length(breaks) == 2)
-      stopifnot(all(sapply(breaks, is.numeric)))
+      breaks <- list(x = NA, y = NA)
```

Comment:
I think using `list(x = NULL, y = NULL)` makes the code below a bit simpler. Then you can just test `is.null(breaks$x)`